Userland/Libraries/LibCrypto/ASN1/DER.cpp

```diff
@@ -234,6 +235,206 @@ ErrorOr<void> Decoder::leave()
     return {};
 }
 
+ErrorOr<void> Encoder::write_tag(Class class_, Type type, Kind kind)
+{
+    auto class_byte = static_cast<u8>(class_);
+    auto type_byte = static_cast<u8>(type);
+    auto kind_byte = static_cast<u8>(kind);
```

Comment:
if these are enums, prefer to_underlying, while specifying the size as u8 at the decl?